src/specifications/profile-resolution/profile-resolution-specml.xml

```diff
@@ -1401,29 +1344,22 @@ control:
       <p>
         <tgt>back-matter</tgt>in the result is produced by combining all objects within
         <src>back-matter</src>in all source catalogs, with the
-        <src>back-matter</src>in the input profile. The merge method and merge structuring directives are ignored. The following requirements MUST be adhered to by the processor:
+        <src>back-matter</src>in the input profile.
       </p>
       <ul>
+        <li><p><req level="must" id="req-backmatter">The output's backmatter MUST be generated by copying in each <src>resource</src> object from the backmatters of the imported catalogs/profiles in top-to-bottom order, then by copying in each <src>resource</src> object from the backmatter of the source profile itself. These objects MUST be processed in the order they are given in their respective documents.</req></p></li>
         <li>
-          <p>Each import&apos;s backmatter is processed in order it was provided in the source profile, then the source profile&apos;s backmatter is processed.</p>
-        </li>
-        <li>
-          <p>Each
-            <src>resource</src>is added to the target in the order given inside the import.
-          </p>
-        </li>
-        <li>
-          <p>If a
+          <p><req level="must" id="req-backmatter-dupe">If a given
             <src>resource</src>has the same
-            <src>uuid</src>as a resource that has already been added, the previous resource is removed, and the more recent one added.
+            <src>uuid</src>as a resource that has already been added, the previous resource MUST be removed, and the more recent one added, unless superceded by other requirements.</req>
           </p>
         </li>
         <li>
-          <p>A resource with a child prop of name:keep and value:always can only be replaced following the above rule by a duplicate that also has the keep always prop.</p>
+          <p><req level="must" id="req-backmatter-keep">A <src>resource</src> with a child <src>prop</src> of <src>name:keep</src> and <src>value:always</src> MUST NOT be replaced by the addition of another <src>resource</src>, unless the new resource also has a child <src>prop</src> of <src>name:keep</src> and <src>value:always</src>.</req></p>
         </li>
       </ul>
-      <p>Tools MAY check for pruning conditions
-        <xref rid="cleanup" />as resources are added as long as the final result is the same as if the pruning had taken place at the end of all resource addition.
+      <p><req level="may" id="req-backmatter-prune">Tools MAY check for pruning conditions
+        <xref rid="cleanup" />as resources are added as long as the final result is the same as if the pruning had taken place at the end of all resource addition.</req>
```

Comment:
I think this will also cause a spacing issue, not sure why I skipped it before.

```suggestion
        <xref rid="cleanup" /> as resources are added as long as the final result is the same as if the pruning had taken place at the end of all resource addition.</req>
```